jinja_to_js/__init__.py

```diff
@@ -86,7 +86,7 @@ def es6_format(dependencies, template_function):
     var __runtime = jinjaToJS.runtime;
     var __filters = jinjaToJS.filters;
     var __globals = jinjaToJS.globals;
-    context = Object.assign({{}}, __globals, context);
+    context = jinjaToJS.createContext(context);
```

Comment:
Reassigning function arguments is generally a Bad Thing ™️ - I would call the function argument `ctx` maybe and then create a local var called `context`.